.github/workflows/publish.yml

```diff
@@ -0,0 +1,29 @@
+name: Publish Python Package
+
+on:
+  push:
```

Comment:
I think it is more frequent that every master commits will have python package publish action. We may consider [on-release-published](https://github.com/sadikkuzu/squiral/blob/main/.github/workflows/python-publish.yml#L6-L8). So we can control pypi publish times via GitHub repo package [releases](https://github.com/serhatbolsu/robotframework-appiumlibrary/releases).

Reply:
Good sugestion, eu fiz essa sugestão no último commit